src/wkwebkit.cs

```diff
@@ -176,6 +176,10 @@ interface WKPreferences : NSCoding {
 
 		[Export ("plugInsEnabled")]
 		bool PlugInsEnabled { get; set; }
+
+		[Mac (10,12,3, onlyOn64 : true)]
```

Comment:
if headers really say 10.2.3 then let's try it and confirm it works (and put a code comment if not). This is a kind of error that's too common in the headers :)